docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+following become true:
+
+1. it fails to perform and disclose its membership-qualifying audit and
+   fifteen (15) months have elapsed since the end of the audit period of its
+   last successful membership-qualifying audit;
+1. its membership-qualifying audit is revoked, rescinded or withdrawn;
+1. its S/MIME certificates are not treated as valid by any Certificate
+   Consumer Member of the SMCWG.
+
+Any Member who believes one of the above circumstances is true of any other
+Member may report it on the SMCWG’s Public Mail List. The SMCWG Chair will then
+investigate, including asking the reported Member for evidence documenting the
+Member’s qualification(s) for membership. If evidence of continued qualification
+for membership is not forthcoming from the reported Member within five (5)
+working days, the SMCWG Chair will announce that such Member is suspended, such
+announcement to include the basis upon which the suspension has been made.
+
+A suspended Member who believes it has re-met the Eligibility to Participate
+criteria as outlined in this charter shall post its evidence to the SMCWG Public
+Mail List or provide evidence to the SMCWG Chair who SHALL post it to the SMCWG
+Public Mail List. The SMCWG Chair will examine the evidence and unsuspend the
+member, or not, by announcement to the SMCWG Public Mail List. A Member’s
+membership will automatically cease six months after the SMCWG Chair’s
+announcement of its suspension if the Member has not re-met the membership
+criteria by that time.
+
+While suspended, a Member may participate in SMCWG Meetings, SMCWG
+Teleconferences, and on the SMCWG’s discussion lists, but may not propose or
+endorse ballots or take part in any form of voting.
+
+Votes cast before the announcement of a Member’s suspension will stand.
+
+### 5. Membership Declaration
+
+In accordance with the IPR Policy, Members that choose to participate in the
+SMCWG MUST declare their participation and SHALL do so prior to participating.
+The Chair of the SMCWG SHALL establish a list for declarations of participation
+and manage it in accordance with the Bylaws, the IPR Policy, and the IPR
+Agreement.
+
+### 6. Voting and Other Organizational Matters
+
+#### 6.1. Voting Structure
+
+The rules described in Bylaw 2.3 and 2.4 SHALL apply to all ballots, including
+Draft Guideline Ballots.
+
+In order for a ballot to be adopted by the SMCWG, two-thirds or more of the
+votes cast by the Certificate Issuers must be in favor of the ballot and more
+than 50% of the votes cast by the Certificate Consumers must be in favor of the
+ballot. At least one member of each class must vote in favor of a ballot for it
+to be adopted. Quorum is the average number of Member organizations (cumulative,
+regardless of Class) that have participated in the previous three (3) SMCWG
+Meetings or Teleconferences (not counting subcommittee meetings thereof). For
+transition purposes, if three (3) meetings have not yet occurred, quorum is
+three (3).
```

Comment:
Especially for the initial calls, this seems like a remarkably low number of participants, especially when it comes to adopting guidelines.

The simplest option is to remove this, and say that Ballots shall not be adopted until at least three meetings have been held. The moving of the the duration of the Chairship should make this reasonable.